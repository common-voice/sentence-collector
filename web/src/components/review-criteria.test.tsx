import React from 'react';
import { render } from '@testing-library/react';

import ReviewCriteria from './review-criteria';

test('should render criteria', () => {
  const { container } = render(<ReviewCriteria />);
  expect(container).toMatchSnapshot();
});
