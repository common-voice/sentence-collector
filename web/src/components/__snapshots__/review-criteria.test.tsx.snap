// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render criteria 1`] = `
<div>
  <span>
    <button
      class="secondary"
    >
      getString-sc-criteria-modal
    </button>
    <div
      class="no-show"
    >
      <div
        class="modal-box"
      >
        <button
          class="close-modal"
        >
          x
        </button>
        <div>
          <h2
            id="review-criteria"
          >
            Review Criteria
          </h2>
          <h3>
            Make sure the sentence meets the following criteria:
          </h3>
          <ol>
            <li>
              The sentence must be spelled correctly.
            </li>
            <li>
              The sentence must be grammatically correct.
            </li>
            <li>
              The sentence must be speakable.
            </li>
            <li>
              If the sentence meets the criteria, click the "yes" button on the right.
            </li>
            <li>
              If the sentence does not meet the above criteria, click the "no" button on the right. If you are unsure about the sentence, you may also skip it and move on to the next one.
            </li>
            <li>
              If you run out of sentences to review, please help us collect more sentences!
            </li>
          </ol>
        </div>
      </div>
    </div>
  </span>
</div>
`;
